appveyor.yml

```diff
@@ -30,7 +30,7 @@ install:
   # install SSL and php
   - ps: Set-Service wuauserv -StartupType Manual
   - cinst -y OpenSSL.Light
-  - cinst -y php
+  - cinst -y php -version 7.1.12
```

Comment:
That makes appveyor work again for now. Problem is that PHP was updated to 7.2, and the folder in which php is located changes (from C:\tools\php71 to C:\tools\php72). The other problem is, that Sulu doesn't seem to be compatible to PHP 7.2 🙈